woocommerce/woocommerce-gutenberg-products-block
# Add dynamic rendering + SSR to the Top Rated Products block

See #586 – This PR updates the Top Rated Products block.

### How to test the changes in this Pull Request:

1. Add a Top Rated Products block to a post before activating this PR
2. Build the PR
3. View your existing post, the block should still render correctly
4. Add a new Top Rated Products block to a post, it should work correctly – show the right products, allow for setting categories in the sidebar, toggling the content, etc.
5. View your post on the frontend, it should display same as the editor preview
